primer/brand
# Add responsive toggling of feature lists in pricing options component

## Summary

Resolves https://github.com/github/primer/issues/3366

Before: Feature lists in PricingOptions weren't collapsible on desktop.
Now: Feature lists in PricingOptions can optionally be collapsed at regular and wide viewports.

No breaking changes, and previous behavior across breakpoints has been preserved:

- Feature lists on narrow viewports are still collapsed by default
- Feature lists from regular viewports and wider, are still initially visible

Also added the ability to immediately expand all feature lists across pricing options on initial selection to aid easier comparison.



## What should reviewers focus on?



- Feature works as described in the description

## Steps to test:



1. [Open this story](https://primer-f9137eb94b-26139705.drafts.github.io/storybook/?path=/story/components-pricingoptions-features--collapsed-features-no-dividers) on a wide viewport
1. Observe feature lists are initially collapsed and accordion toggle appears 
1. Toggle the 'What's included' accordion on any feature list to be expanded
1. Observe ALL feature lists are now expanded



## Supporting resources (related issues, external links, etc):

- Slack thread

## Contributor checklist:


## Reviewer checklist:


## Screenshots:


https://github.com/user-attachments/assets/e1f64362-1016-4e42-9028-c06ba7617e94
